Upload separate artifacts for installer CRDs and installer CRs (#2555)
kyma-project/test-infra: development/tools/cmd/githubrelease/README.md

| **--kymaInstallerCluster**     |    No    | The string value with the name of the Kyma cluster configuration file. It defaults to `kyma-installer-cluster.yaml`.
| **--kymaConfigLocal**          |    No    | The string value with the name of the Kyma local configuration file. It defaults to `kyma-config-local.yaml`.
| **--kymaInstallerLocal**       |    No    | The string value with the name of the Kyma local configuration file. It defaults to `kyma-installer-local.yaml`.

> | **--components**               |    No    | The string value with the name of the file with list of components installed by the installer. It defaults to `components.yaml`.

Reviewer: ```suggestion
| **--components**               |    No    | Name of the file containing the list of components to install. It defaults to `components.yaml`.
```

Author: This would change the parameter description for only this parameter and hence wont be uniform with rest of description so I would like to leave it that way.
